controllers/k8ssandra/seeds.go

```diff
@@ -0,0 +1,79 @@
+package k8ssandra
+
+import (
+	"context"
+	"github.com/go-logr/logr"
+	cassdcapi "github.com/k8ssandra/cass-operator/apis/cassandra/v1beta1"
+	api "github.com/k8ssandra/k8ssandra-operator/apis/k8ssandra/v1alpha1"
+	"github.com/k8ssandra/k8ssandra-operator/pkg/utils"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// findSeeds queries for pods labeled as seeds. It does this for each DC, across all
+// clusters.
+func (r *K8ssandraClusterReconciler) findSeeds(ctx context.Context, kc *api.K8ssandraCluster, logger logr.Logger) ([]corev1.Pod, error) {
```

Comment:
Shouldn't this exclude pods from the same datacenter as the target updated seed? As in, return map[datacenter][]corev1.Pod for example.

Reply:
I am going mark this resolved based on the conversation with @adejanovski where he said having redundant seeds shouldn't be a problem. @burmanm please feel free to reopen if you want to further discuss.